app/views/snps/show.html.erb

```diff
@@ -7,7 +7,7 @@
     <table class="table table-striped">
       <tr>
         <td><b>Name</b></td>
-        <td><%=@snp.name%></td>
+        <td><%=@snp.name%> </td>
```

Comment:
Is there a reason for the added space? Also, in general, it's better to add a bit of space within the angle brackets to improve readability:
```erb
<td><%= @snp.name %></td>
```